examples/pipelines/pipeline_utils.py

```diff
@@ -1,37 +1,15 @@
-"""General kfp helpers"""
+"""General pipeline utils"""
```

Comment:
Any reason the pipeline utils moved out of Express to here?

Reply:
the reason is that we wanted to move the `kfp` dependencies out of express and have it only in the `requirements.txt` the `example/pipelines` level